user-access-commons/src/main/java/no/unit/nva/database/RoleDb.java

```diff
@@ -65,7 +67,17 @@ public void setPrimaryHashKey(String primaryHashKey) throws InvalidEntryInternal
     @DynamoDBRangeKey(attributeName = PRIMARY_KEY_RANGE_KEY)
     @Override
     public String getPrimaryRangeKey() {
-        return getType();
+        return this.primaryRangeKey;
+    }
+
+    @Override
+    public void setPrimaryRangeKey(String primaryRangeKey) throws InvalidEntryInternalException {
```

Comment:
Primary Range key set according to the practices described in DynamoDb book by Alex de Brie.